### ISSUE TITLE
 train curve and validation curve

### ISSUE THREAD

**Issue by kankanzheli** [open-mmlab/mmdetection#9650]
### What's the feature?

I want to show the data sets and models visually through curves. According to my study, train curve and validation curve can do this. Is there any way to output the curves of both?

**Comment by RangiLyu** [comment]
If you mean the loss curve, you can turn on the tensorboard logger to visualize it. https://github.com/open-mmlab/mmdetection/blob/1b7d778da310eccff95bb8e401f5845a03b17ba9/configs/_base_/default_runtime.py#L7
And if you want to show the val loss, you need to add a val workflow(`workflow = [('train', 1), ('val', 1)] `) at here:
https://github.com/open-mmlab/mmdetection/blob/1b7d778da310eccff95bb8e401f5845a03b17ba9/configs/_base_/default_runtime.py#L16
